audio/out/ao_aaudio.c

```diff
@@ -0,0 +1,248 @@
+/*
```

Comment:
We use 4 spaces in mpv. Take a look at some other files and follow the style if possible.